content/webapp/components/ExhibitionCaptions/ExhibitionCaptions.tsx

```diff
@@ -87,70 +176,139 @@ const Stop: FC<{ stop: Stop }> = ({ stop }) => {
-    >
-      <TitleTombstone>
-        <h2>
-          {stop.number ? `${stop.number}. ` : ''}
-          {stop.title}
-        </h2>
-        <em>
-          <PrismicHtmlBlock html={stop.tombstone} />
-        </em>
-      </TitleTombstone>
-      <CaptionTranscription>
-        <Caption>
-          {stop.image?.contentUrl && (
-            <Space v={{ size: 'l', properties: ['margin-bottom'] }}>
-              <PrismicImageWrapper>
-                <PrismicImage image={stop.image} sizes={{}} quality={`low`} />
-              </PrismicImageWrapper>
+    <>
+      {standaloneTitle.length > 0 && (
+        <div className="container">
+          <Space
+            v={{
+              size: 'xl',
+              properties: ['margin-bottom'],
+            }}
+          >
+            <Divider color={`pumice`} isKeyline={true} />
+          </Space>
+          <div className="flex flex--wrap">
+            <Tombstone />
+            {/* This empty Tombstone is needed for correct alignmennt of the standaloneTitle */}
+            <Space
+              h={{
+                size: 'm',
+                properties: ['margin-left'],
+                negative: true,
+              }}
+              v={{ size: 'l', properties: ['margin-bottom'] }}
+            >
+              <StandaloneTitle>{standaloneTitle}</StandaloneTitle>
             </Space>
+          </div>
+        </div>
+      )}
+      <Space v={{ size: 'xl', properties: ['margin-bottom'] }}>
+        <ConditionalWrapper
+          condition={hasContext}
+          wrapper={children => (
+            <ContextContainer
+              backgroundColor={isFirstStop ? 'white' : 'cream'}
+              backgroundShade={isFirstStop ? 'base' : 'light'}
+              hasPadding={!isFirstStop}
+            >
+              {children}
+            </ContextContainer>
           )}
-          <PrismicHtmlBlock html={stop.caption} />
-        </Caption>
-        {transcriptionText && (
-          <Transcription>
-            <h3>Audio transcript</h3>
-            <div id="transcription-text">
-              <PrismicHtmlBlock
-                html={transcriptionText as prismicT.RichTextField}
-              />
-            </div>
-            {hasShowFullTranscriptionButton && (
-              <ButtonSolid
-                colors={themeValues.buttonColors.greenTransparentGreen}
-                ariaControls="transcription-text"
-                ariaExpanded={isFullTranscription}
-                clickHandler={() => {
-                  setIsFullTranscription(!isFullTranscription);
-                }}
-                text={
-                  isFullTranscription
-                    ? 'Hide full transcript'
-                    : 'Read full transcript'
-                }
-              />
-            )}
-          </Transcription>
-        )}
-      </CaptionTranscription>
-    </Space>
+        >
+          <div className="flex flex--wrap container">
```

Comment:
There's something about having `flex` and `container` on the same element that is setting off a distant alarm bell for me, but looking at what `.container` does, I'm not sure why now – I've probably misremembered something